--- origsrc/PDL-2.007/Graphics/PLplot/Makefile.PL	2013-05-12 15:09:47.000000000 -0500
+++ src/PDL-2.007/Graphics/PLplot/Makefile.PL	2014-11-11 00:02:09.390224600 -0600
@@ -79,6 +79,11 @@ foreach my $libdir (
     $plplot_lib_path = $libdir;
     $ENV{LD_LIBRARY_PATH} .= ":$libdir";
     last;
+  } elsif (-e "$libdir/$libname.a") {
+    $plplot_lib_path = $libdir;
+    $ENV{LD_LIBRARY_PATH} .= ":$libdir";
+    $libname = $libname.a;
+    last;
   } elsif (-e "$libdir/$libname_static") {
     $plplot_lib_path = $libdir;
     $ENV{LD_LIBRARY_PATH} .= ":$libdir";
@@ -197,7 +202,8 @@ $hash{'META_MERGE'}   = {provides => {PD
 $hash{'LIBS'}      = [$libs];
 # add this to allow one to use the /ops/tools/lib plplot libs even when
 # another plplot is installed in /usr/local/lib. D. Hunt 6/4/2004
-$hash{'LDDLFLAGS'} = $^O eq 'darwin' ? " -bundle -undefined dynamic_lookup -L$plplot_lib_path " : " -shared -L$plplot_lib_path -L/usr/local/lib ";
+$hash{'LDDLFLAGS'} = $^O eq 'darwin' ? " -bundle -undefined dynamic_lookup -L$plplot_lib_path " : " -shared -L$plplot_lib_path  " . $Config{lddlflags};
+
 
 $hash{'INC'}      .= " $plplot_include_path";
 $hash{'clean'}     = {'FILES'  =>
--- origsrc/PDL-2.007/Lib/GIS/Proj/Makefile.PL	2013-05-12 15:09:47.000000000 -0500
+++ src/PDL-2.007/Lib/GIS/Proj/Makefile.PL	2014-11-10 23:59:39.458649000 -0600
@@ -17,7 +17,7 @@ BEGIN
    # Generic descripton of how to optionally add this package to the PDL Tree:
    $package_name = "PDL::GIS::Proj";
    $lib_name = "Proj";
-   $find_libs = [ 'libproj.so', 'libproj.a' ];
+   $find_libs = [ 'libproj.so', 'libproj.dll.a', 'libproj.a' ];
    $find_incs = [ 'proj_api.h' ];
    $config_flag = 'WITH_PROJ';
    $config_libs = 'PROJ_LIBS';
--- origsrc/PDL-2.007/Lib/Transform/Proj4/Makefile.PL	2013-05-12 15:09:47.000000000 -0500
+++ src/PDL-2.007/Lib/Transform/Proj4/Makefile.PL	2014-11-10 23:59:59.414790400 -0600
@@ -41,7 +41,7 @@ BEGIN
    # Generic descripton of how to optionally add this package to the PDL Tree:
    $package_name = "PDL::Transform::Proj4";
    my $lib_name = "Proj4";
-   my $find_libs = [ 'libproj.so', 'libproj.a' ];
+   my $find_libs = [ 'libproj.so', 'libproj.dll.a', 'libproj.a' ];
    my $find_incs = [ 'proj_api.h' ];
    my $config_flag = 'WITH_PROJ';
    my $config_libs = 'PROJ_LIBS';
