--- origsrc/PDL-2.4.11/Graphics/PLplot/Makefile.PL	2012-01-02 17:56:28.000000000 -0600
+++ src/PDL-2.4.11/Graphics/PLplot/Makefile.PL	2012-07-12 04:22:55.974404200 -0500
@@ -79,6 +79,11 @@ foreach my $libdir (
     $plplot_lib_path = $libdir;
     $ENV{LD_LIBRARY_PATH} .= ":$libdir";
     last;
+  } elsif (-e "$libdir/$libname.a") {
+    $plplot_lib_path = $libdir;
+    $ENV{LD_LIBRARY_PATH} .= ":$libdir";
+    $libname = $libname.a;
+    last;
   } elsif (-e "$libdir/$libname_static") {
     $plplot_lib_path = $libdir;
     $ENV{LD_LIBRARY_PATH} .= ":$libdir";
@@ -197,7 +202,7 @@ $hash{'META_MERGE'}   = {provides => {PD
 $hash{'LIBS'}      = [$libs];
 # add this to allow one to use the /ops/tools/lib plplot libs even when
 # another plplot is installed in /usr/local/lib. D. Hunt 6/4/2004
-$hash{'LDDLFLAGS'} = $^O eq 'darwin' ? " -bundle -undefined dynamic_lookup -L$plplot_lib_path " : " -shared -L$plplot_lib_path -L/usr/local/lib ";
+$hash{'LDDLFLAGS'} = " -L$plplot_lib_path " . $Config{lddlflags};
 
 $hash{'INC'}      .= " $plplot_include_path";
 $hash{'clean'}     = {'FILES'  =>
--- origsrc/PDL-2.4.11/IO/GD/Makefile.PL	2012-01-02 17:56:28.000000000 -0600
+++ src/PDL-2.4.11/IO/GD/Makefile.PL	2012-07-12 04:32:02.582668400 -0500
@@ -15,7 +15,7 @@ BEGIN
     $ppfile = "GD.pd";
     $package_name = "PDL::IO::GD";
     $lib_name = "GD";
-    $find_libs = [ 'libgd.so', 'libgd.a', 'bgd.dll' ];
+    $find_libs = [ 'libgd.so', 'libgd.dll.a', 'libgd.a', 'bgd.dll' ];
     $find_incs = [ 'gd.h' ];
     $config_flag = 'WITH_GD';
     $config_libs = 'GD_LIBS';
@@ -76,7 +76,7 @@ BEGIN
                 $lib_path = $libdir;
                 $found = 1;
                 # The lib name is different on windows, so we need to adjust the LIBS, below:
-                $linkname = ( $find_lib =~ /bgd.dll/ ) ? 'bgd' : 'gd';
+                $linkname = ( $find_lib =~ /bgd.dll$/ ) ? 'bgd' : 'gd';
             }
             last if $found;
         }
--- origsrc/PDL-2.4.11/Lib/GIS/Proj/Makefile.PL	2012-01-02 17:56:28.000000000 -0600
+++ src/PDL-2.4.11/Lib/GIS/Proj/Makefile.PL	2012-07-10 23:16:52.987421100 -0500
@@ -17,7 +17,7 @@ BEGIN
    # Generic descripton of how to optionally add this package to the PDL Tree:
    $package_name = "PDL::GIS::Proj";
    $lib_name = "Proj";
-   $find_libs = [ 'libproj.so', 'libproj.a' ];
+   $find_libs = [ 'libproj.so', 'libproj.dll.a', 'libproj.a' ];
    $find_incs = [ 'proj_api.h' ];
    $config_flag = 'WITH_PROJ';
    $config_libs = 'PROJ_LIBS';
--- origsrc/PDL-2.4.11/Lib/Transform/Proj4/Makefile.PL	2012-01-02 17:56:28.000000000 -0600
+++ src/PDL-2.4.11/Lib/Transform/Proj4/Makefile.PL	2012-07-10 23:28:21.512494800 -0500
@@ -16,7 +16,7 @@ BEGIN
    # Generic descripton of how to optionally add this package to the PDL Tree:
    $package_name = "PDL::Transform::Proj4";
    $lib_name = "Proj4";
-   $find_libs = [ 'libproj.so', 'libproj.a' ];
+   $find_libs = [ 'libproj.so', 'libproj.dll.a', 'libproj.a' ];
    $find_incs = [ 'proj_api.h' ];
    $config_flag = 'WITH_PROJ';
    $config_libs = 'PROJ_LIBS';
