--- origsrc/PDL-2.4.3/Lib/FFTW/Makefile.PL	2004-01-12 14:00:06.000000000 -0600
+++ src/PDL-2.4.3/Lib/FFTW/Makefile.PL	2008-03-17 21:09:50.781250000 -0500
@@ -149,7 +149,7 @@
 		# make a list of libs from the libstring
 		my @libs = split ' ',$double_lib;
 		grep s/\-l//g, @libs;
-		grep $_ = "lib$_.a", @libs;
+		grep $_ = "lib$_.dll.a", @libs;
 		my %libs;
 		my $lib;
 		foreach $libdir(@libdirs){
--- origsrc/PDL-2.4.3/Lib/GSL/Makefile.PL	2004-01-12 14:00:06.000000000 -0600
+++ src/PDL-2.4.3/Lib/GSL/Makefile.PL	2008-03-17 21:09:50.781250000 -0500
@@ -33,7 +33,7 @@
 ($GSL_includes, $GSL_libs)  = get_gsl_libs();
 
 # Version check
-my $MINVERSION = "1.3";
+my $MINVERSION = "1.03";
 my $version = `gsl-config --version`;
 chomp $version;
 
--- origsrc/PDL-2.4.3/perldl.conf	2006-08-16 21:59:51.000000000 -0500
+++ src/PDL-2.4.3/perldl.conf	2008-03-17 21:09:50.796875000 -0500
@@ -91,7 +91,7 @@
 # There are problems with the build on OS-X, so we turn it off by default
 # for such systems
 #
-	WITH_3D => $^O eq "darwin" ? 0 : undef,
+	WITH_3D => $^O eq "cygwin" ? 0 : undef,
 
 #
 #  For Mesa 3.2
@@ -182,7 +182,7 @@
 # Whether or not to build the PDL::GIS::Proj module
 # 0 -> don't use
 # true -> force use
-        WITH_PROJ => undef,
+        WITH_PROJ => 0,
         PROJ_LIBS => undef,
         PROJ_INC => undef,
         
